### PR TITLE
Integrate Review comment API with UI

### PR DESCRIPTION
**Pull request by jadmsaadaot**
*Issue #:*
https://github.com/bcgov/met-public/issues/316

*Description of changes:*

- Integrate Review comment API with UI
- Improve view comments listing page
- improve review comment page

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Review by tiagograf-AOT — APPROVED** [review]

File: met-web/src/routes/AuthenticatedRoutes.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ import SurveyListing from 'components/survey/listing';
 import CreateSurvey from 'components/survey/create';
 import SurveyFormBuilder from 'components/survey/building';
 import SurveySubmit from 'components/survey/submit';
-import CommentReview from 'components/comments/CommentReview';
+import CommentReview from 'components/comments/admin/review/CommentReview';
```
Comment (tiagograf-AOT) [inline met-web/src/routes/AuthenticatedRoutes.tsx:11]: why the admin? we already mixed up public/internal stuff anyway

File: met-web/src/services/commentService/index.tsx [hunk 2]
```diff
@@ -29,13 +29,14 @@ export const getComment = async (commentId: number): Promise<Comment> => {
...
 
-interface UpdateCommentRequest {
-    id: string;
-    survey_id: number;
+interface ReviewCommentRequest {
+    status_id: number;
+    comment_id: number;
 }
-export const UpdateComment = async (data: UpdateCommentRequest): Promise<Comment> => {
+export const ReviewComment = async ({ comment_id, status_id }: ReviewCommentRequest): Promise<Comment> => {
```
Comment (tiagograf-AOT) [inline met-web/src/services/commentService/index.tsx:36]: in the frontend we use camelCase commentId and the backend the snake case comment_id, quite confusing, maybe lets have a task to standardize

**Review by saravanpa-aot — APPROVED** [review]

File: met-web/src/routes/AuthenticatedRoutes.tsx (hunk 1 above)
Comment (jadmsaadaot) [reply met-web/src/routes/AuthenticatedRoutes.tsx:11]: Comments pages are comment review, comment listing (admin), comment text listing (admin), and comment text listing (public). When I was creating the file structure it made sense to separate them like that under the comments folder. What would you suggest instead?

File: met-web/src/services/commentService/index.tsx (hunk 2 above)
Comment (jadmsaadaot) [reply met-web/src/services/commentService/index.tsx:36]: problem is camelCase is kinda the standard in JS and snake case in python. Choosing one for both maybe is choosing ones standard over the other. Could be wrong.

**Pull request merged**